src/index.js

```diff
@@ -176,6 +176,16 @@ if (!PLATFORM.global.System || !PLATFORM.global.System.import) {
   };
 } else {
   PLATFORM.eachModule = function(callback) {
+    if (System.registry) { // SystemJS >= 0.20.x
+      for (let [k, m] of System.registry.entries()) {
```

Comment:
Form the [registry API](https://github.com/systemjs/systemjs/blob/ec39166a8e89a0fe3048cddc63d45c5580deffea/docs/system-api.md#systemjsregistry) I do not see how to avoid the iterable. But could switch from `for...of` to `Array.from`+`for`.